fluster/test.py

```diff
@@ -56,7 +56,15 @@ def __init__(
         super().__init__(test_vector.name)
 
     def _test(self) -> None:
-        output_filepath = os.path.join(self.results_dir, self.test_vector.name + ".yuv")
+        if self.decoder.name == "FDK-AAC":
+            output_filepath = os.path.join(
+                self.results_dir, self.test_vector.name + ".wav"
+            )
+        else:
+            output_filepath = os.path.join(
+                self.results_dir, self.test_vector.name + ".yuv"
+            )
+
```

Comment:
All this should be on the other commit: https://github.com/fluendo/fluster/pull/87/commits/ef10c6004bc36684afb3fbd82ce6bd8213ac65b3

Reply:
Do we need the extension? no IMHO. At the end, the output_filepath is only a temporal file. We can use for yuv and for wav (and for rbga):
```
output_filepath = os.path.join(self.results_dir, self.test_vector.name + 'raw')
```